lib/googleauth/signet.rb

```diff
@@ -84,7 +84,11 @@ def fetch_access_token! options = {}
       end
 
       def notify_refresh_listeners
-        listeners = @refresh_listeners || []
+        if defined? @refresh_listeners
+          listeners = @refresh_listeners
+        else
+          listeners = []
+        end
```

Comment:
Looks like Rubocop is flagging this. Prefer:
```ruby
listeners =
  if defined? @refresh_listeners
    @refresh_listeners
  else
    []
  end
```
Or alternately:
```ruby
listeners = defined?(@refresh_listeners) ? @refresh_listeners : []
```

Reply:
Done. Looks like rubocop is happy now:

```
diff --git a/lib/googleauth/signet.rb b/lib/googleauth/signet.rb
index 0b9f93d..db0dfa8 100644
--- a/lib/googleauth/signet.rb
+++ b/lib/googleauth/signet.rb
@@ -84,11 +84,7 @@ module Signet
       end

       def notify_refresh_listeners
-        if defined? @refresh_listeners
-          listeners = @refresh_listeners
-        else
-          listeners = []
-        end
+        listeners = defined?(@refresh_listeners) ? @refresh_listeners : []
         listeners.each do |block|
           block.call self
         end

google-auth-library-ruby$ rubocop
Inspecting 21 files
.....................

21 files inspected, no offenses detected
```